src/features/bees/components/Bee.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useContext } from "react";
+import { useActor } from "@xstate/react";
+
+import coop from "assets/nfts/chicken_coop.png";
+
+import { GRID_WIDTH_PX } from "features/game/lib/constants";
+import { Context } from "features/game/GameProvider";
+import { Section } from "lib/utils/hooks/useScrollIntoView";
+
+export const Bee: React.FC = () => {
+  const { gameService, selectedItem } = useContext(Context);
+  const [
+    {
+      context: { state },
+    },
+  ] = useActor(gameService);
+
+  return (
+    <>
+      {state.inventory["Chicken Coop"] && (
```

Comment:
checking chicken coop in inventory for display of bee hive?

Reply:
Corrected! thanks for pointing it out